README.md

```diff
@@ -32,3 +32,11 @@ If you'd like to make an enhancement to the application, please see [the contrib
 
 ### iOS
 - On iOS you can use [testflight to install app](https://testflight.apple.com/join/Rxzv6JpC).
+
+## Trademarks ##
+
+PhotoPrism® is a registered trademark of Michael Mayer. You may use it as required to describe 
+our software, run your server, for educational purposes, but not for offering commercial 
+goods, products, or services without prior written permission. In other words, please ask.
```

Comment:
*Let's keep it short and add a link to our contact page if users have any questions (same for the app dialog):*

~~In other words, please ask.~~

Feel free to reach out if you have questions:

https://photoprism.app/contact

~~See also https://github.com/photoprism/photoprism#trademarks~~

Reply:
done, if that is all, I'll click the merge button?

Reply:
Thank you!